satpy/tests/reader_tests/test_mersi_l1b.py

```diff
@@ -237,7 +248,7 @@ def _get_geo_data(num_scans, rows_per_scan, num_cols, prefix):
             xr.DataArray(
                 da.ones((num_scans * rows_per_scan, num_cols), chunks=1024),
                 attrs={
-                    "Slope": np.array([1.] * 1), "Intercept": np.array([0.] * 1),
+                    "Slope": np.array([1.] * 1, dtype=np.float32), "Intercept": np.array([0.] * 1, dtype=np.float32),
```

Comment:
Angles should remain in float64 right?

Reply:
Yes, `Longitude` and its intercept and slope seem to be `float64`. Fixed in https://github.com/pytroll/satpy/pull/2976/commits/3bb6db5894e74e7a00af4ffe7f99b9243590c025